src/material-experimental/mdc-menu/menu.scss

```diff
@@ -1 +1,60 @@
-// TODO: MDC core styles here.
+@import '@material/menu-surface/mixins';
+@import '@material/list/mixins';
+@import '@material/list/variables';
+@import '../../lib/core/style/menu-common';
+@import '../../lib/core/style/button-common';
+
+@include mdc-menu-surface-core-styles($query: structure);
+
+.mat-mdc-menu-content {
+  @include mdc-list-base_($query: structure);
```

Comment:
The main one adds a ton of CSS that we aren't using.

Reply:
I see, I guess because it probable includes all kinds of CSS for different list variants that we're not concerned with here. Can you add a comment so its obvious why we're using the private mixins? Hopefully at some point in the future MDC can refactor their mixins so there's a public one we can consume that gives just the basic list styles without all the extras.